packages/commons/src/repositories/ReadModelRepository.ts

```diff
@@ -23,7 +23,9 @@ export type GenericCollection<T> = Collection<{
   metadata: Metadata;
 }>;
 
-export type EServiceCollection = GenericCollection<EService>;
+// Uncomment this line when all services can read models with date type fields
+// export type EServiceCollection = GenericCollection<EService>;
```

Comment:
I would add a link to the issue also in this comment, as in the other two, to clarify why